source/_integrations/shelly.markdown

```diff
@@ -119,6 +119,17 @@ Not all devices support all input events. You can check on [Shelly API Reference
 
 </div>
 
+## Services
+
+### Service `shelly.ota_update`
+
+Triggers an over-the-air update for selected Shelly device(s). At least one of `area_id` or `device_id` has to be provided. This could be easily done via the target selector on UI.
```

Comment:
This was the indication for me to go to the parent PR and leave a comment on the implementation. This should be implemented as an entity service.

That way, it can use a target (meaning, entity_id, area_id or device_id and even a mix of it, is accepted).